### ISSUE TITLE
TypeError in Craft 3.7.28 when saving a form (1.5.4)

### ISSUE THREAD

**Issue by RyanRoberts** [verbb/formie#701]
**Description**

On saving a form I get the following error (in logs) which prevents the form being saved:

```
TypeError: verbb\formie\services\Forms::validatePage(): Argument #2 ($page) must be of type verbb\formie\models\FieldLayoutPage, craft\models\FieldLayoutTab given, called in /Users/ryan/Git/client/vendor/verbb/formie/src/services/Forms.php on line 617 and defined in /Users/ryan/Git/client/vendor/verbb/formie/src/services/Forms.php:652
Stack trace:
#0 /Users/ryan/Git/client/vendor/verbb/formie/src/services/Forms.php(617): verbb\formie\services\Forms->validatePage(Object(verbb\formie\elements\Form), Object(craft\models\FieldLayoutTab), false)
#1 /Users/ryan/Git/client/vendor/verbb/formie/src/elements/Form.php(1436): verbb\formie\services\Forms->validateFormFields(Object(verbb\formie\elements\Form))
#2 /Users/ryan/Git/client/vendor/verbb/formie/src/services/Forms.php(100): verbb\formie\elements\Form->validate()
#3 /Users/ryan/Git/client/vendor/verbb/formie/src/controllers/FormsController.php(156): verbb\formie\services\Forms->saveForm(Object(verbb\formie\elements\Form))
#4 [internal function]: verbb\formie\controllers\FormsController->actionSave()
#5 /Users/ryan/Git/client/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#6 /Users/ryan/Git/client/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#7 /Users/ryan/Git/client/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('save', Array)
#8 /Users/ryan/Git/client/vendor/craftcms/cms/src/web/Application.php(287): yii\base\Module->runAction('formie/forms/sa...', Array)
#9 /Users/ryan/Git/client/vendor/craftcms/cms/src/web/Application.php(596): craft\web\Application->runAction('formie/forms/sa...', Array)
#10 /Users/ryan/Git/client/vendor/craftcms/cms/src/web/Application.php(266): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#11 /Users/ryan/Git/client/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#12 /Users/ryan/Git/client/public/index.php(22): yii\base\Application->run()
#13 /Users/ryan/.composer/vendor/laravel/valet/server.php(234): require('/Users/ryan/Git...')
#14 {main}
```

**Steps to reproduce**
1. Open form in Craft
2. Hit save

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.4
- Craft version: 3.7.28
- Multi-site: No

**Additional context**

Currently this is running locally in Valet

**Comment by janeschindler** [comment]
I'm also getting this when I try to access forms via graphql.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.5](https://github.com/verbb/formie/releases/tag/1.5.5)
